actions/filter_list_or_object_MOD.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   name: 'Filter List/Object',
+  displayName: 'Filer List or Object',
```

Comment:
Filer? you mean Filter

Reply:
i would install a spell checker on vsc and check everything just in case
engrish is not really any of our strong suites